api/helpers/gamestate/publish-game-state.js

```diff
@@ -101,6 +101,8 @@ module.exports = {
         playedBy,
         pNum,
         // Conditionally included properties if truthy
+        ...(gameState.cardPlayed && {playedCardId: gameState.cardPlayed.id}),
```

Comment:
GameState's attribute is called `playedCard`
```suggestion
        ...(gameState.playedCard && {playedCardId: gameState.playedCard.id}),
```